docs/siem/detections/prebuilt-rules/rule-details/elastic-endpoint.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[elastic-endpoint-prebuilt-rule]]
-=== Elastic Endpoint
+=== Elastic Endpoint Security
```

Comment:
We may have to loop back around and change these post-merge. See my previous comment about Elastic Endpoint Security for an explanation.

Reply:
This will be OK as it refers to a prebuilt rule name that's already finalised (for 7.9 at least).